packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/sdk.md

```diff
@@ -0,0 +1,75 @@
+### 1: Initiate password recovery
+
+To begin the password recovery flow, the user clicks the **Forgot Password** button on the login page. Then they select **Email** as the authenticator to be used for password recovery on the subsequent page and click **Submit**. Okta then sends an email to their email address matching the Forgot Password template that was altered earlier.
+
+<div class="common-image-format">
+
+![Screenshot of email sent to user](/img/advanced-use-cases/custom-pwd-recovery-custom-email.png)
```

Comment:
Please use the following section in the Dev Docs style guide for guidance and update your image description: https://github.com/okta/okta-developer-docs/wiki/Style-Guide#define-alt-text-for-images-and-diagrams

Reply:
Have added alt text to all images